src/lib/readmeAPIFetch.ts

```diff
@@ -475,7 +479,7 @@ export async function fetchMappings(this: CommandClass['prototype']): Promise<Ma
 /**
  * Fetches the schema for the current route from the OpenAPI description for ReadMe API v2.
  */
-export async function fetchSchema(this: CommandsThatSyncMarkdown): Promise<SchemaObject> {
+export async function fetchSchema(this: CommandsThatSyncMarkdown) {
```

Comment:
we not running with isolatedDeclarations?

Reply:
ah we probably should now, i'll enable that

Reply:
ugh it's a lot more work to enable it in this repo than i have time for atm. will look into it in a follow-up